### DISCUSSION TITLE
Can custom attributes be added to menu shortcut links?

### DISCUSSION THREAD

**Discussion by bekopharm** [McShelby/hugo-theme-relearn#645] [Q&A] [answered]
I've a bunch of shortcuts in the menu that are _mostly_ social media links and I'd like to add the `rel=me` attribute to the links.

I know that I can use HTML in `name` but that does not influence how the link itself is generated.

What I want in the end is basically something like this:

`<a class="padding" rel="me" href="https://social.tchncs.de/@bekopharm"><i class="openwebicons openwebicons-mastodon"></i> Mastodon</a>`

> https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/me

**Comment by McShelby** [comment, marked as answer]
Sadly, the theme does not offer such customization. As of now you have three choices:

- add JavaScript into your `custom-footer.html` to set the attributes
- modify existing theme files (this will cause you headaches when you want to update the theme later)
- don't do it at all

**Comment by McShelby** [comment]
A fourth thing could  be to file a PR. Idea is, to allow for arbitrary attributes configurable in the config.toml for each link.

**Comment by bekopharm** [comment]
Ah yes, sadly. Kinda expected this but it is what it is. JS is not really an option because tools that parse this usually do not execute scripts.

The good thing is that it doesn't matter where that link is so an invisible link in the header does the job too. The human readable part doesn't require this. It just adds more noise when added twice.

Anyway, thanks for confirming!
